hack/tools/create_istio.sh

```diff
@@ -46,25 +51,6 @@ istio_config() {
     kubectl label namespace default istio-injection=enabled --overwrite
```

Comment:
why do we need to label the default namespace?

Reply:
will remove